tests/code_layout/test_sipfiles_uptodate.sh

```diff
@@ -25,12 +25,14 @@ for module in "${modules[@]}"; do
         outdiff=$(./scripts/sipify.pl -p python/${module}/auto_additions/${pyfile}.temp $header | diff python/$sipfile.in -)
         if [[ -n "$outdiff" ]]; then
           echo " *** SIP file not up to date: $sipfile"
+          echo " $outdiff "
```

Comment:
Let's not forget to remove this before it's merged

Reply:
This is already merged